packages/ui/src/domains/Marketing/LandingPage/CoreServices.tsx

```
@@ -0,0 +1,64 @@
+		},
+	];
+
+	return (
+		<section className="py-16 px-4">
+			<div className="max-w-6xl mx-auto">
+				<h2 className="text-3xl font-bold text-center mb-2">Core Services</h2>
+				<div className="w-24 h-1 bg-blue-600 mx-auto mb-12"></div>
+				<div className="grid grid-cols-1 md:grid-cols-3 gap-8">
+					{services.map((service, index) => (
+						<div
+							key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)